src/main/java/uk/gov/hmcts/darts/usermanagement/controller/SecurityGroupController.java

```diff
@@ -27,7 +27,7 @@ public class SecurityGroupController implements SecurityGroupApi {
     @SecurityRequirement(name = SECURITY_SCHEMES_BEARER_AUTH)
     @Authorisation(contextId = ANY_ENTITY_ID, globalAccessSecurityRoles = ADMIN)
     public ResponseEntity<List<SecurityGroupWithIdAndRole>> adminSecurityGroupsGet(Integer courthouse) {
```

Comment:
Do we need to do something with this optional query param? Looks like it should be renamed to be `Integer courthouseId` to match the api generated interface

Reply:
Filtering by courthouse may be an imminent requirement, so suggest leaving it here but we don't need to do anything with it at this stage.

Reply:
Filtering by courthouse may be an imminent requirement, so suggest leaving it here but we don't need to do anything with it at this stage.